specification/batchai/resource-manager/Microsoft.BatchAI/2017-09-01-preview/BatchAI.json

```diff
@@ -1950,7 +1950,7 @@
             "$ref": "#/definitions/EnvironmentSetting"
           },
           "title": "Additional environment variables to be passed to the job.",
-          "description": "Batch AI services sends the following environment variables for all jobs: AZ_BATCHAI_INPUT_id, AZ_BATCHAI_OUTPUT_id, AZ_BATCHAI_NUM_GPUS_PER_NODE, For distributed TensorFlow jobs, following additional environment variables are set by the Batch AI Service: AZ_BATCHAI_PS_HOSTS, AZ_BATCHAI_WORKER_HOSTS"
+          "description": "Batch AI services sets the following environment variables for all jobs: AZ_BATCHAI_INPUT_id, AZ_BATCHAI_OUTPUT_id, AZ_BATCHAI_NUM_GPUS_PER_NODE, For distributed TensorFlow jobs, following additional environment variables are set by the Batch AI Service: AZ_BATCHAI_PS_HOSTS, AZ_BATCHAI_WORKER_HOSTS"
```

Comment:
I see that the phrase "Batch AI" was removed in a lot of other places but not here, is that intentional?

Reply:
Also no period at end of sentence.

Reply:
Batch AI was removed before Cluster, Job and File Server. Like it was "Batch AI Cluster", now it's "Cluster".  But Batch AI service remains.
added the period